Source/WebCore/editing/InsertTextCommand.cpp

```diff
@@ -230,8 +231,17 @@ void InsertTextCommand::doApply()
 
     setEndingSelectionWithoutValidation(startPosition, endPosition);
 
-    // Handle the case where there is a typing style.
-    if (RefPtr<EditingStyle> typingStyle = document().selection().typingStyle()) {
+    CheckedRef frameSelection = document().selection();
```

Comment:
Is this CheckedRef safe? (since the code below seems to update layout)

Reply:
Good catch, the code below does indeed update layout via `Element::computedStyle`.

Fixed.